tests/test_measurements_collapse.py

```diff
@@ -207,24 +205,24 @@ def test_measurement_result_parameters_multiple_qubits(backend):
     backend.assert_allclose(final_state, target_state)
 
 
-@pytest.mark.skip(reason="no way of currently testing this")
+@pytest.mark.skip(reason="this has to be updated for density matrices")
```

Comment:
So this is not testing all scenarios?

Reply:
This is an old test that was requiring a configuration that we do not support anymore, it might be kept by updating it to make use of the density matrix, but that's not straightforward to me